src/app/productConfig/productConfigModal.tpl.html

```diff
@@ -21,11 +21,19 @@ <h4 class="modal-title" translate>
   <button class="btn btn-success" ng-click="$dismiss()">Keep Browsing</button>
 </div>
 
+
 <form novalidate name="$ctrl.itemConfigForm" ng-submit="$ctrl.addToCart()" ng-if="!$ctrl.giftSubmitted">
   <div class="modal-body pt0 p">
     <div class="container-fluid">
       <div class="panel panel-default panel-plain mt">
         <div class="panel panel-default give-modal-panel">
+
+          <div ng-if="$ctrl.error" ng-switch="$ctrl.error.indexOf('already in the cart') !== -1" class="alert" role="alert"
+               ng-class="{'alert-warning': $ctrl.error.indexOf('already in the cart') !== -1, 'alert-danger': $ctrl.error.indexOf('already in the cart') === -1}">
+            <span ng-switch-when="true" translate>This gift has already been added. Please proceed to checkout.</span>
```

Comment:
I might add something about you can edit it in the cart (maybe their intentions were to give a different amount/frequency than however it got added to the cart previously). Maybe someone gave you specific requirements, that was just my thought.